Tools/autotest/arduplane.py

```diff
@@ -5474,4 +5514,5 @@ def disabled_tests(self):
             "LandingDrift": "Flapping test. See https://github.com/ArduPilot/ardupilot/issues/20054",
             "InteractTest": "requires user interaction",
             "ClimbThrottleSaturation": "requires https://github.com/ArduPilot/ardupilot/pull/27106 to pass",
+            "CatapultTakeoff": "Known bug. See https://github.com/ArduPilot/ardupilot/issues/27147",
```

Comment:
if it is passing now we can remove this line

Reply:
Ah, indeed. Thanks.